src/scene/materials/material.js

```diff
@@ -728,6 +728,9 @@ class Material {
         return this.parameters;
     }
 
+    /**
+     * Clears all cached shader variants from the material and mesh instances that reference it.
```

Comment:
we should add `@protected` tag on it - allowing derived classes to use it, but not making a public API